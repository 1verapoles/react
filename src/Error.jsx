import React from 'react'
import './Error.css'
import img from './assets/images/404.png'

function Error() {
  return (
    <div className="container">
      <div className="error-wrapper">
        <div className="error-inner">
          <img src={img} alt="error" />
          <div className="error-text">Sorry, smth went <br /> wrong((
          </div>
        </div>
      </div>
    </div>
  )
}


export default Error
