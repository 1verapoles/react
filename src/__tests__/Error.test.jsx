import React from 'react';
import { render } from '@testing-library/react';
import Error from '../Error';

describe('Testing Error component', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<Error />);

    expect(asFragment()).toMatchSnapshot();
  });
});
