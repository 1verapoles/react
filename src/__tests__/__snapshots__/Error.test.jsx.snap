// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Error component renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="error-wrapper"
    >
      <div
        class="error-inner"
      >
        <img
          alt="error"
          src="[object Object]"
        />
        <div
          class="error-text"
        >
          Sorry, smth went 
          <br />
           wrong((
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
